### PR TITLE
Added docs links for IDEs & added furthur readings for frameworks & tools

### PR DESCRIPTION
**Pull request by Seek4samurai**
## Description
Added Docs & forums links of respective IDEs in IDEs section page. Also added some helpful article links in the further reading section of the frameworks page & IDE page.

Added some articles in the further reading sections that are well written and can help the reader.

### CODE REVIEW

File: src/content/developers/docs/ides/index.md [hunk 1]
```diff
@@ -57,4 +63,6 @@ Most established IDEs have built plugins to enhance the Ethereum development exp
 
 ## Further reading {#further-reading}
 
+- [Blockchain development tools](https://www.alchemy.com/overviews/20-blockchain-development-tools) - Alchemy's top 20 blockchain development tools
```
Comment (minimalsm) [inline src/content/developers/docs/ides/index.md:66]: While seems like an ok list, this doesn't seem all that relevant to people looking at IDEs 🤔 

```suggestion
```

File: src/content/developers/docs/frameworks/index.md [hunk 2]
```diff
@@ -95,6 +95,9 @@ Before diving into frameworks, we recommend you first read through our introduct
 
 ## Further reading {#further-reading}
 
+- [Guide to the Ethereum stack](https://www.freecodecamp.org/news/full-stack-ethereum-development/) - Freecodecamp's guide to full Ethereum stack
+- [More about dapps](https://www.investopedia.com/terms/d/decentralized-applications-dapps.asp) - Investopedia on the term dapps
+
```
Comment (minimalsm) [inline src/content/developers/docs/frameworks/index.md:100]: I'm not sure these are relevant to development frameworks.

File: src/content/developers/docs/ides/index.md [hunk 3]
```diff
@@ -12,9 +12,15 @@ If you're looking to fiddle with code before you [set up a local development env
...
+- [Docs](https://remix-ide.readthedocs.io/en/latest/#)
+- [Gitter](https://gitter.im/ethereum/remix)
+
 **[ChainIDE](https://chainide.com/)** - **_A cloud-based multi-chain IDE_**
 
-**[Replit (Solidity Starter)](https://replit.com/@replit/Solidity-starter-beta)** - **_A customizable development environment for Ethereum with hot reloading, error checking, and first-class testnet support_**
+- [Docs](https://chainide.gitbook.io/chainide-english-1/)
+- [Help forum](https://forum.chainide.com/)
+
+**[Replit (Solidity Starter - Beta)](https://replit.com/@replit/Solidity-starter-beta)** - **_A customizable development environment for Ethereum with hot reloading, error checking, and first-class testnet support_**
```
Comment (minimalsm) [inline src/content/developers/docs/ides/index.md:23]: Thanks 😘

File: src/content/developers/docs/frameworks/index.md (hunk 2 above)
Comment (corwintines) [reply src/content/developers/docs/frameworks/index.md:100]: I agree. Going to remove

**Review by corwintines — APPROVED** [review]
Thanks, @Seek4samurai!

I removed some content based on @minimalsm comments, and added docs for Replit to follow some of your other additions!

**Pull request merged**